src/Microsoft.AspNetCore.Blazor/Routing/Router.cs

```diff
@@ -31,6 +31,11 @@ public class Router : IComponent, IDisposable
         /// </summary>
         [Parameter] private Assembly AppAssembly { get; set; }
 
+        /// <summary>
+        /// Gets or sets the route that should be used when no components with are found with the requested route.
```

Comment:
> components with are found with

typo: double `with`

Reply:
Fixed it and ironically put another typo in the commit message.